semver/README.md

```diff
@@ -0,0 +1,58 @@
+# A parser for Semantic Version 2.0
+
+## The What
+
+This is a parser for [semver 2.0 spec](https://semver.org/).
+In addition the spec adherence, it supports version "tags" such as "beta3" or "rc.2" - this would allow implementers to implement version comparison not only with major/minor/patch versions but with the numbers of beta/rc versions.
+Therefore, it would be possible to compare and see that "1.2.3-rc2" > "1.2.3-beta23" > "1.2.3-beta8" > "1.2.3-alpha8"  
+
+## The How
+
+An example implementation in C# can be found [here](https://github.com/myarichuk/ReleaseTools/tree/master/src/Parser.ConventionalCommit).
+
+## Regarding Antlr4cs
+
+In case the implementation of the grammar needs to be used with [Antlr4cs](https://github.com/tunnelvisionlabs/antlr4cs) C# targets, the following changes to the lexer will make the grammar compatible.
+
+```antlrv4
+fragment A : [aA];
+fragment B : [bB];
+fragment C : [cC];
+fragment D : [dD];
+fragment E : [eE];
+fragment F : [fF];
+fragment G : [gG];
+fragment H : [hH];
+fragment I : [iI];
+fragment J : [jJ];
+fragment K : [kK];
+fragment L : [lL];
+fragment M : [mM];
+fragment N : [nN];
+fragment O : [oO];
+fragment P : [pP];
+fragment Q : [qQ];
+fragment R : [rR];
+fragment S : [sS];
+fragment T : [tT];
+fragment U : [uU];
+fragment V : [vV];
+fragment W : [wW];
+fragment X : [xX];
+fragment Y : [yY];
+fragment Z : [zZ];
+
+fragment ALPHA: A L P H A;
+fragment BETA: B E T A;
+fragment RC: (R C) | (R E L E A S E (DASH | DOT) C A N D I D A T E);
+fragment SNAPSHOT: S N A P S H O T;
+fragment PREVIEW: P | P R E | P R E V I E W;
+fragment DEV: D E V | D E V E L | D E V E L O P M E N T;
+fragment MILESTONE: M T | M I L E S T O N E;
+fragment DAILY: D A I L Y;
+fragment NIGHTLY: N I G H T L Y;
+fragment BUILD: B L D | B U I L D;
+fragment TEST: T E S T;
+fragment EXPERIMENTAL: E X P E R I M E N T A L;
+
+```
```

Comment:
Sorry, but I don't consider this info useful, we don't have such info in other case insensitive grammars (check [`sql`](https://github.com/antlr/grammars-v4/tree/master/sql) for example). It's well-known practice to use case insensitive fragments. Also, I think there is a small number of outdated Antlr4cs runtime.

```suggestion
```

Reply:
Makes sense, I will remove this section in the readme